homework/index.js

```diff
@@ -28,9 +30,37 @@
     });
     return elem;
   }
-
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    // eslint-disable-next-line prefer-template
+    const li = createAndAppend('li', ul, { id: 'item' });
```

Comment:
Your generated HTML contains multiple occurrences of the `item` **id**. This violates the HTML rule that **id** attributes should be unique, i.e. occurs once only in an HTML document.

Reply:
Yes you are right, I was confused about the whole homework and forget this point